src/core/comparable.mli

```diff
@@ -5,6 +5,5 @@ module type S = sig
 
   include Equatable.S with type t := t
 
-  val compare:t Comparator.t
-end
-
+  val compare: t Comparator.t
```

Comment:
We have a top level namespace file called `Collection`, so say if the user wants to use `Sequence` he will probably will have to alias `Collection.Sequence`. But I think for consistency I will change all the modules to start with a `B`.